policyengine_us/variables/gov/states/ky/tax/income/credits/personal/ky_personal_tax_credits.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class ky_personal_tax_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Kentucky personal tax credits"
+    unit = USD
+    documentation = (
+        "https://apps.legislature.ky.gov/law/statutes/statute.aspx?id=53500"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.KY
+
```

Comment:
Please restructure the other credits to be the amount of the credit. We can skip eligibility variables. They can stay as person-level (with `head_or_spouse` limitation); `adds` will sum it to tax unit.
```suggestion
    adds = ["ky_personal_tax_credit_blind", "ky_personal_tax_credit_aged", "ky_personal_tax_credit_military"]
```